infrabox/test/api/job_api_test.py

```diff
@@ -0,0 +1,183 @@
+from os import getcwd, stat, remove
+
+from pyinfraboxutils.token import encode_job_token
+from pyinfraboxutils.storage import storage
+from temp_tools import TestClient
+from test_template import ApiTestTemplate
+
+
+class JobApiTest(ApiTestTemplate):
+
+    url_ns = 'api/job'
+
+    #def test_uuid_validation(self):
+    #   pass
+    #   #TODO
+
+    def test_job(self):
+        filename = 'test_file1.json'
+        file_path = getcwd() + '/' + filename
+        filesize = stat(file_path).st_size
+        TestClient.execute("""TRUNCATE source_upload""")
```

Comment:
move all truncates to the base class so we don't have to repeat them in each test